### PR TITLE
[Doc] Add *Redis sink connector guide*

### PR DESCRIPTION
**Pull request by Anonymitaet**
Fix https://github.com/apache/pulsar/issues/5015

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@tuteng could you please help review? Thank you

File: site2/docs/io-redis-sink.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+| `redisDatabase` | int|true|0  | The Redis database to connect to. |
+| `clientMode` |String| false|Standalone | The client mode when interacting with Redis cluster. <br><br>Below are the available options: <br><li>Standalone<br><li>Cluster |
+| `autoReconnect` | boolean|false|true | Whether the Redis client automatically reconnect or not. |
+| `requestQueue` | int|false|2147483647 | The maximum number of queued requests to Redis. |
+| `tcpNoDelay` |boolean| false| false | Whether to enable TCP with no delay or not. |
+| `keepAlive` | boolean|false | false |Whether to enable a keepalive to Redis or not. |
+| `connectTimeout` |long| false|10000 | The time to wait before timing out when connecting in milliseconds. |
+| `operationTimeout` | long|false|10000 | The time before an operation is marked as timed out in milliseconds . |
+| `batchTimeMs` | int|false|1000 | The Redis operation time in milliseconds. |
+| `batchSize` | int|false|1000 | The batch size of writing to Redis database. |
```
Comment (tuteng) [inline site2/docs/io-redis-sink.md]: The default value for field `batchSize` is 200

File: site2/docs/io-redis-sink.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+The configuration of the Redis sink connector has the following properties.
+
+
+
+### Property
+
+| Name | Type|Required | Default | Description 
+|------|----------|----------|---------|-------------|
+| `redisHosts` |String|true|" " (empty string) | A comma-separated list of Redis hosts to connect to. |
+| `redisPassword` |String|true|" " (empty string) | The password used to connect to Redis. |
```
Comment (tuteng) [inline site2/docs/io-redis-sink.md]: Field `redisPassword` is optional.

Comment (Anonymitaet) [reply site2/docs/io-redis-sink.md]: OK, I've udpated.

File: site2/docs/io-redis-sink.md (hunk 1 above)
Comment (Anonymitaet) [reply site2/docs/io-redis-sink.md]: OK, I've updated.

**Comment by tuteng** [comment]
run cpp tests

**Comment by Anonymitaet** [comment]
retest

**Comment by Anonymitaet** [comment]
run cpp tests
run integration tests
run java8 tests

**Comment by Anonymitaet** [comment]
run cpp tests
run integration tests

**Comment by Anonymitaet** [comment]
run cpp tests
run integration tests
run java8 tests

**Comment by Anonymitaet** [comment]
run integration tests
run java8 tests

**Comment by Anonymitaet** [comment]
run java8 tests

**Comment by Anonymitaet** [comment]
Thank you @aahmed-se

**Pull request merged**